src/views/Transactions.vue

```diff
@@ -93,8 +71,7 @@ export default defineComponent({
 
         await filterTransactions(page.value)
       } catch (e) {
-        console.error(e.message)
-        handleError(e)
+        handleError(e as Error)
```

Comment:
catch (e: Error) {
    handleError(e)
}

Reply:
Catch clause variable type annotation must be 'any' or 'unknown' if specified.Vetur(1196)